kernel/arch/dreamcast/util/vmu_fb.c

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Please provide a correct header comment here.

Also, please remember that KOS' core kernel code is not GPL. I will not be able to accept this PR for the kernel if you intend for this to be GPL licensed.

Reply:
I'm too used to SPDX headers, I just forgot to remove it.